tests/test_blotter.py

```diff
@@ -58,7 +58,9 @@ def setUpClass(cls):
         cls.env.write_data(equities_data={
             24: {
                 'start_date': cls.sim_params.trading_days[0],
-                'end_date': cls.sim_params.trading_days[-1]
+                'end_date': cls.env.next_trading_day(
```

Comment:
the price/NaN issue.  we need the asset to live to the day after the last trading day, so that we can get prices on the last trading day.